src/Model/PhasorDynamics/Governor/Tgov1/Tgov1Impl.hpp

```diff
@@ -150,7 +160,7 @@ namespace GridKit
         // Initial mechanical = initial electric torque
         if (signals_.template isAssigned<Tgov1InternalVariables::PM>())
         {
-          p0 = y_[2]; //<- generator needs to be initialized first
+          p0 = y_[2]; ///<- generator needs to be initialized first
         }
```

Comment:
Again, I'd revert this change.

Reply:
Addressed in [2b0573e](https://github.com/ORNL/GridKit/pull/232/commits/2b0573e630404a68fe52f7cdb6036c5bfb2bfe1b)